src/components/input-date-picker/input-date-picker.tsx

```diff
@@ -597,11 +604,13 @@ export class InputDatePicker
                   activeRange={this.focusedInput}
                   endAsDate={this.endAsDate}
                   headingLevel={this.headingLevel}
+                  //t9n props are used here to forward the messages only.
                   intlNextMonth={this.intlNextMonth}
```

Comment:
Shouldn't these be forwarded with messagesOverrides? Applies to other places too.

Reply:
These need to be removed as we are passing `messageOverrides` here anyways.

Reply:
`input-date-picker` is not a `t9nComponent` which means when the user sets `intl-*` props or `messageOverrides` they won't be in sync and it is the `date picker's job to merge those and keep them in sync which is why we need to prase them.

Reply:
Gotcha. Yeah maybe we can just remove documentation for this component.